packages/memtrace_viewer_with_deps/memtrace_viewer_with_deps.v0.14.0/opam

```diff
@@ -0,0 +1,56 @@
+opam-version: "2.0"
+maintainer: "opensource@janestreet.com"
+authors: ["Jane Street Group, LLC <opensource@janestreet.com>"]
+homepage: "https://github.com/janestreet/memtrace_viewer"
+bug-reports: "https://github.com/janestreet/memtrace_viewer/issues"
+dev-repo: "git+https://github.com/janestreet/memtrace_viewer.git"
+doc: "https://ocaml.janestreet.com/ocaml-core/latest/doc/memtrace_viewer/index.html"
+license: "MIT"
+build: [
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: [
+  [make "uninstall"]
+]
```

Comment:
Could `dune build --profile=release @memtrace_viewer/install` be used directly for the build phase? Also could you remove the `install` and `remove` phase and instead add
```
  ["cp" "memtrace_viewer.install" "."]
```
to the build phase?
`dune install` requires ocamlfind at runtime and is not recommended in opam and `dune uninstall` is unnecessary with opam 2.0.